modules/agar/src/test/ts/webdriver/RealMouseTest.ts

```diff
@@ -79,3 +79,62 @@ UnitTest.asynctest('RealMouseTest', (success, failure) => {
     success();
   }, failure);
 });
+
+UnitTest.promiseTest('RealMouseTest promise based variant', async () => {
```

Comment:
When using BDD tests we should always be using `describe`, `it`, etc... instead of the legacy qunit style.

- [ ] Convert the new test to BDD style